### PR TITLE
Add DD_API_SECURITY_ENABLED flag

### PR DESCRIPTION
**Pull request by estringana**
### Reviewer checklist
- [ ] Test coverage seems ok.
- [ ] Appropriate labels assigned.

### CODE REVIEW

File: appsec/src/extension/configuration.h [hunk 1]
```diff
@@ -63,7 +63,7 @@ extern bool runtime_config_first_init;
     CONFIG(CUSTOM(STRING), DD_APPSEC_AUTOMATED_USER_EVENTS_TRACKING, "safe", .parser = dd_parse_automated_user_events_tracking)       \
     CONFIG(STRING, DD_APPSEC_HTTP_BLOCKED_TEMPLATE_HTML, "")                                                                          \
     CONFIG(STRING, DD_APPSEC_HTTP_BLOCKED_TEMPLATE_JSON, "")                                                                          \
-    CONFIG(DOUBLE, DD_API_SECURITY_REQUEST_SAMPLE_RATE, "0.1")
+    CONFIG(BOOL, DD_API_SECURITY_ENABLED, "true")
```
Comment (bwoebi) [inline appsec/src/extension/configuration.h:66]: Either you don't do get_global, or you mark this as a system INI.

Comment (estringana) [reply appsec/src/extension/configuration.h]: Done

**Review by Anilm3 — CHANGES_REQUESTED** [review]
The sample rate should not be removed, the new variable should be present in addition to the sample rate.

**Comment by estringana** [comment]
> The sample rate should not be removed, the new variable should be present in addition to the sample rate.

Done

**Review by Anilm3 — APPROVED** [review]
lgtm

**Pull request merged**